source/dotnet/Library/AdaptiveCards.Templating/AdaptiveCardsTemplateVisitor.cs

```diff
@@ -611,10 +612,13 @@ public static string Expand(string unboundString, IMemory data, bool isTemplated
                 return unboundString;
             }
 
-            var options = new Options
```

Comment:
can you remove https://github.com/microsoft/AdaptiveCards/pull/5964/files#diff-75a381746eb6256f934ad5e0d008c5d87268fd74cd97162e6cd385f69c5ddd7fL614-L617 instead?

options are already initialized here
https://github.com/microsoft/AdaptiveCards/pull/5964/files#diff-75a381746eb6256f934ad5e0d008c5d87268fd74cd97162e6cd385f69c5ddd7fL124-L127

Reply:
`Expand` is `static` though, which I think is causing the issue.

Reply:
https://github.com/microsoft/AdaptiveCards/pull/5964/files/f7d9938a7333bbe5adf8ce1e88d13f2277534569#diff-75a381746eb6256f934ad5e0d008c5d87268fd74cd97162e6cd385f69c5ddd7fR615-R621. These lines seem to be redundant. I guess it won't hurt.